client/components/forms/form-button/index.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+import { omit } from 'lodash';
+import React, { Children } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@wordpress/components';
+
+class FormButton extends React.Component {
+	static defaultProps = {
+		isSubmitting: false,
+		isPrimary: true,
+		type: 'submit',
+	};
+
+	getDefaultButtonAction = () => {
+		return this.props.isSubmitting
+			? this.props.translate( 'Saving…' )
+			: this.props.translate( 'Save Settings' );
+	};
+
+	render() {
+		const { children, className, isPrimary, ...props } = this.props,
+			buttonClasses = classNames( className, 'form-button', 'button', {
+				'is-borderless': !! this.props.borderless
```

Comment:
Added handling for `borderless` prop that we use in a few cases. Using `is-borderless` similar to https://github.com/Automattic/woocommerce-services/pull/2361/files#diff-dc83733ab281e3cf1cb35835dbedfc2abdd540bf59d78e0f4e4e55863c3edab4R229.